database/migrations/2020_06_08_140727_update__s_k_config_in_production.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+use App\Evaluation;
+use App\Sponsor;
+use Illuminate\Database\Migrations\Migration;
+
+class UpdateSKConfigInProduction extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // Only run in production
```

Comment:
This will also run on staging, right? Since it is env production.

Reply:
yes. we should see warnings about no SK